lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -561,12 +571,60 @@ void OpenCLFunction::execute() {
 
       if (auto *SI = dyn_cast<SplatInst>(&I)) {
         // Pass the splat as a parameter.
-        setKernelArg(kernel, numArgs + 1, SI->getValue());
+        setKernelArg(kernel, ++numArgs, SI->getValue());
       } else if (auto *EPI = dyn_cast<ElementPowInst>(&I)) {
         // Pass the exp as a parameter.
-        setKernelArg(kernel, numArgs + 1, EPI->getExp());
+        setKernelArg(kernel, ++numArgs, EPI->getExp());
       }
 
+      if (isQuantized) {
+        int32_t destOffset = I.getOperand(0).first->getType()->getOffset();
```

Comment:
use names instead of getOperand(0) etc.

Reply:
Thx! But i cant call getDest() here as instruction I was not casted into any specific instruction, and it does not have a getDest() member function